### PR TITLE
sql/sqlspec: add comment fields to table, column and index structs

### PR DESCRIPTION
**Pull request by masseelch**
Please note, that the changes to the Marshal test is necessary until I implement the Marshal method for comments.

### CODE REVIEW

**Comment by masseelch** [comment]
Please note, that the changes to the Marshal test is necessary until I implement the Marshal method for comments.

File: sql/internal/specutil/convert.go [hunk 1]
```diff
@@ -106,6 +106,9 @@ func Table(spec *sqlspec.Table, parent *schema.Schema, convertColumn ConvertColu
 		}
 		tbl.Indexes = append(tbl.Indexes, i)
 	}
+	if spec.Comment != "" {
+		tbl.Attrs = append(tbl.Attrs, &schema.Comment{Text: spec.Comment})
```
Comment (rotemtam) [inline sql/internal/specutil/convert.go:110]: can we use `SetComment` from the fluent API?

File: sql/internal/specutil/convert.go [hunk 2]
```diff
@@ -117,6 +120,9 @@ func Column(spec *sqlspec.Column, conv ConvertTypeFunc) (*schema.Column, error)
 			Null: spec.Null,
 		},
 	}
+	if spec.Comment != "" {
+		out.Attrs = append(out.Attrs, &schema.Comment{Text: spec.Comment})
```
Comment (rotemtam) [inline sql/internal/specutil/convert.go:124]: same

**Review by rotemtam — COMMENTED** [review]
Great start. Tiny comment

File: internal/integration/hclsqlspec/hclsqlspec_test.go [hunk 3]
```diff
@@ -328,8 +328,9 @@ func TestMarshalTopLevel(t *testing.T) {
 	h, err := hcl.MarshalSpec(c)
 	require.NoError(t, err)
 	require.EqualValues(t, `column "column" {
-  null = true
-  type = varchar(255)
+  null    = true
+  type    = varchar(255)
+  comment = ""
```
Comment (a8m) [inline internal/integration/hclsqlspec/hclsqlspec_test.go:333]: Why the `comment` is not omitted in this case?

Comment (masseelch) [reply internal/integration/hclsqlspec/hclsqlspec_test.go:333]: Because I want to add the `Marshal` in a second PR. This is needed to make the test pass.

**Review by a8m — APPROVED** [review]

**Pull request merged**